tests/API/JsonSerializeTest.php

```diff
@@ -152,6 +155,10 @@ public static function init()
         self::$OBJ_SORT->setOrder(Sort\Order::Descending)
             ->setField("price");
 
+        self::$OBJ_RESTRICT_NAVIGATION = new RestrictNavigation();
+        self::$OBJ_RESTRICT_NAVIGATION->setCount(2);
```

Comment:
Chain the set name here (100% personal preference, but it makes it look neater)

->setCount()->setName()